takwimu/templates/takwimu/_includes/search/table.html

```diff
@@ -72,7 +72,7 @@
                         {% if not result.data_point.value.indicators %}
```

Comment:
We're indexing and searching `TopicBlock` body, which is a `RichTextField`, but on the display table we're showing `IndicatorBlock`...

Reply:
@kilemensi I added the TopicBlock.body to the indicator